Gemfile

```diff
@@ -2,16 +2,16 @@ source 'https://rubygems.org'
 
 ruby RUBY_VERSION # Needed to consider & solve for Ruby version requirements
 
-gem 'berkshelf'
+gem 'berkshelf', '~> 4.3.5'
 gem 'chef', '~> 12.0'
-gem 'chefspec'
+gem 'chefspec', '~> 7.1'
 gem 'codeclimate-test-reporter', '~> 1.0.0'
-gem 'foodcritic'
-gem 'oneview-sdk', '~> 4.5.1'
+gem 'foodcritic', '~> 11.3'
```

Comment:
Actually it is not the best approach.

I'll lower the version binding, it is not safe to leave it causing issues as new versions appears in the new rubies. (same as rubocop rules)